py/selenium/webdriver/common/action_chains.py

```diff
@@ -27,6 +30,8 @@
 from .actions.wheel_input import WheelInput
 from .utils import keys_to_typing
 
+any_device = Union[PointerInput, KeyInput, WheelInput]
```

Comment:
the agreement for type names is they are in CamelCase and start with uppercase letter

Reply:
> the agreement for type names is they are in CamelCase and start with uppercase letter

Fixing this.